android/conftest.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+from browserstack.local import Local
+import os
+from appium import webdriver
+import json
+from jsonmerge import merge
+CONFIG_FILE = os.environ['CONFIG_FILE'] if 'CONFIG_FILE' in os.environ else 'config/single.json'
```

Comment:
is this `'config/single.json'` correct? I can see single.json inside run-single-test folder